ddtrace/tracer/option.go

```diff
@@ -1258,6 +1261,15 @@ func setHeaderTags(headerAsTags []string) bool {
 	return true
 }
 
+// WithContribStats opens up a channel of communication between tracer and contrib libraries
+// for submitting stats from contribs to Datadog via the tracer's statsd client
+// It is enabled by default but can be disabled with `WithContribStats(false)`
+func WithContribStats(enabled bool) StartOption {
+	return func(c *config) {
+		c.contribStats = enabled
+	}
+}
+
```

Comment:
What's the rationale behind making this configurable? It's of course possible that we'll end up creating a channel that isn't used if someone doesn't use the feature, but that's not going to be a major problem as long as we also clean it up.

Reply:
Yes, just in case we don't want to "lock away" the option to disable it. I can remove it if you think it's more confusing than helpful, though.

Reply:
(Looks like this is resolved now)